app/models/ability.rb

```diff
@@ -73,6 +86,21 @@ def initialize(user)
       can? :download, f
     end
 
+    can :read, StacksMediaStream do |f|
+      user_in_location, _rule = f.location_rights(user.location)
+      f.restricted_by_location? && user_in_location
+    end
```

Comment:
Is this functionally different than https://github.com/sul-dlss/digital_stacks_rails/blob/master/app/models/ability.rb#L66-L70 given https://github.com/sul-dlss/digital_stacks_rails/blob/master/app/models/ability.rb#L66-L70?